ui/app/components/job-status/failed-or-lost.hbs

```diff
@@ -8,13 +8,9 @@
   <div class="failed-or-lost-links">
     {{#if @supportsRescheduling}}
       <span>
-        <span
-          class="tooltip multiline text-center"
-          role="tooltip"
-          aria-label="Allocations that have been rescheduled, on another node if possible, due to failure or manual restart"
-        >
+        <Hds::TooltipButton @text="Allocations that have been rescheduled, on another node if possible, due to failure or manual restart">
```

Comment:
Side-effect to update the tooltips here from custom-rolled ones to the official Helios tooltips.